packages/api/src/APIClient.ts

```diff
@@ -78,16 +90,12 @@ export default class APIClient {
 				Authorization: `Bearer ${token}`,
 				'Api-Version': 'alpha',
 				'Content-Type': 'application/json',
+				'User-Agent': this.clientId,
```

Comment:
It would be nice to include the sdk details as well as the clientId. 
For instance: https://github.com/freshbooks/freshbooks-python-sdk/blob/master/freshbooks/client.py#L73

Maybe for consistency we should go with: `"FreshBooks nodejs sdk/<SDK VERSION> client_id <self.client_id>`

Reply:
Makes sense, will fix that